src/hotspot/cpu/aarch64/macroAssembler_aarch64.cpp

```diff
@@ -66,6 +65,8 @@
 #include "opto/output.hpp"
 #endif
 
+#include <sys/types.h>
```

Comment:
Is that import even used? I can't spot it.

Reply:
This was simply to correct the include order of the already included includes. `#include <sys/types.h>` appeared above the precompiled header include. The rule is to have system includes below the ordinary includes with one new empty line in-between. 

Might very well be that this include is not used.